EazyE2E.Console/Config.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Configuration;
```

Comment:
Config class in the consuming application to read its config settings from its config file.